root/forms/DialogResult.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * @flow strict
+ * Copyright (C) 2024 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import escapeClosingTags from '../utility/escapeClosingTags.js';
+
+component DialogResult(
+  result: string,
+) {
+  const script = (
+    'window.dialogResult = ' +
+    escapeClosingTags(result) +
+    ';'
+  );
+  return (
+    <html>
+      <head>
+        <script dangerouslySetInnerHTML={{__html: script}} />
```

Comment:
Can you please remind (in commit message) what Perl equivalent does it replace? And the origin(s) of values that `result` (used to build `script`) is going to take?

Reply:
I split out 3104264712ca138e9a0888334d96a3cf3bfb398c into its own commit and tried to improve the message. Hope that looks better.